src/main/apps.js

```diff
@@ -525,6 +525,13 @@ function removeSourceDirectory(source) {
     return fs.remove(config.getAppsRootDir(source));
 }
 
+const migrateStoreIfNeeded = () => {
+    const oldStore = new Store({ name: 'pc-nrfconnect-core' });
+    if (oldStore.size > 0 && store.size === 0) {
+        store.store = JSON.parse(JSON.stringify(oldStore.store));
+    }
+};
```

Comment:
The old store is not deleted nor timestamp compared which will lead to the last state of the old store every time.